plugin-jps/src/com/intellij/plugins/thrift/jps/ThriftBuilder.java

```diff
@@ -46,8 +46,7 @@ public ExitCode build(CompileContext context,
                         ModuleLevelBuilder.OutputConsumer outputConsumer) throws ProjectBuildException, IOException {
     ThriftConfig thriftConfig = ThriftConfig.getSettings(context.getProjectDescriptor().getProject());
 
-    final String compiler = VfsUtil.urlToPath(thriftConfig.getCompilerPath());
```

Comment:
nit: would be good to remove `import com.intellij.openapi.vfs.VfsUtil;` from imports since it is not used anymore.